File: first.js
const checker = (email) => {
  const splittedEmail = email.split("@")[1];
  if (splittedEmail !== "gmail.com") {
    return true;
  }
  return "you must use an official email";
};

console.log(checker("iyayiemmanuel1@insight7.io"));

File: first.py
def check(email): 
    """
    It checks if the email is a gmail email or not.
    
    :param email: The email address to be checked
    :return: True
    """
    splittedEmail = email.split('@')[1]
    if splittedEmail != 'gmail.com':
        return True
    return 'Invalid email'

print(check('iyayiemmanuel1@gmail.com'))